components/resourceDetails/index.jsx

```diff
@@ -129,6 +140,26 @@ export default function ResourceDetails({
                 Download
               </DownloadLink>
             </ActionMenuItem>
+            <ActionMenuItem>
+              <Tooltip
+                PopperProps={{
+                  disablePortal: true,
+                }}
+                open={tooltipOpen}
+                disableFocusListener
+                disableHoverListener
+                disableTouchListener
+                title="Copied"
+                data-testid={TESTCAFE_RESOURCE_ADDRESS_TOOLTIP}
+              >
+                <Typography
```

Comment:
This should actually be a button, not just text:

See the `Delete Button` as an example: https://github.com/inrupt/pod-browser/blob/2fcfe4583c8092f48bb495c6ef9f0c2746c84a84/components/deleteButton/index.jsx#L120-L126

Or the Download button, which is the simpler one and more preferred: https://github.com/inrupt/pod-browser/blob/914d0406cea0a9fa699886b1a20f829fc4a4094c/components/downloadLink/index.jsx#L62

Reply:
Done!